src/frontend/App/hooks/useHentAlleGjenbrukbareVilkårsvurderinger.ts

```diff
@@ -0,0 +1,36 @@
+import { useCallback, useState } from 'react';
+import { RessursStatus } from '../typer/ressurs';
+import { useApp } from '../context/AppContext';
+
+interface Props {
+    gjenbrukbareVilkårsvurderinger: string[];
+    hentAlleGjenbrukbareVilkårsvurderinger: (behandlingId: string) => Promise<void>;
+}
+
+export const useHentAlleGjenbrukbareVilkårsvurderinger = (): Props => {
```

Comment:
```suggestion
export const useHentGjenbrukbareVilkårsvurderinger = (): Props => {
```
Kommenterte dette i backend-pren også men vanlig praksis er at man alltid henter alle forekosmter på det man spør om, så "alle" blir redundant

Reply:
Endret